css/at-rules/font-face.json

```diff
@@ -783,6 +783,403 @@
             }
           }
         },
+        "font-width": {
+          "__compat": {
+            "spec_url": "https://drafts.csswg.org/css-fonts/#descdef-font-face-font-width",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "18.4"
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "condensed": {
+            "__compat": {
+              "spec_url": "https://drafts.csswg.org/css-fonts/#valdef-font-width-condensed",
+              "support": {
+                "chrome": {
+                  "version_added": false
```

Comment:
Why are we adding all the subfeatures here, when `css.at-rules.font-face.font-stretch` doesn't have any? I wouldn't add these proactively here.

Reply:
Removed with f71c3a587c7b31b8396834d56b5f8780529cea3b